chagrade/settings/base.py

```diff
@@ -193,6 +211,7 @@
 else:
     # Allows us to use with django-oauth-toolkit on localhost sans https
     SESSION_COOKIE_SECURE = False
+    SESSION_COOKIE_SAMESITE = None
```

Comment:
Purpose of this setting?

Reply:
I can't remember. I'll be sure to put more detail in commit messages with obscure changes like this. For now, it seems to work without it just fine.